test/test.go

```diff
@@ -85,3 +93,32 @@ func CreateClientIP(c *gin.Context) {
 	c.Request.Header = make(map[string][]string)
 	c.Request.Header.Add("X-Forwarded-For", "0.0.0.0")
 }
+
+func Walk(nodes []Node, f func(Node) bool) {
```

Comment:
Obrigado por isso @davidsmedeiros 💯 . Sugestão pra ficar mais claro o objetivo 😀
```suggestion
func WalkThroughXml(nodes []Node, f func(Node) bool) {
```